src/components/TextInput/TextInput.jsx

```diff
@@ -36,37 +39,42 @@ const TextInput = ({
   error,
   data,
   maxLength,
+  id,
+  name,
+  style,
```

Comment:
Why did you include `TextInput` component in these changes?

Reply:
WDYM?

Reply:
Oh.
It's because it has to support `name` in general and `id` specifically for `htmlFor` to work.